schema.sql

```diff
@@ -1,5 +1,16 @@
 /* Database schema to keep the structure of entire database. */
 
+-- CREATE TABLE animals (
+--     name varchar(100)
+-- );
+
+CREATE DATABASE vet_clinic;
+
 CREATE TABLE animals (
-    name varchar(100)
+  id INTEGER,
+  name TEXT,
+  date_of_birth DATE,
+  escape_attempts INTEGER,
+  neutered BOOLEAN,
+  weight_kg DECIMAL
 );
```

Comment:
- [ ]  Kindly add VARCHAR for all your columns as it determines the length of characters this is best practice 👍. 

- [ ] [OPTIONAL] you can make the id column not null and autoincremented
`id int NOT NULL IDENTITY(1,1)`

- [ ] [OPTIONAL] you can mark the id column as the primary key
`PRIMARY KEY (id)`

Reply:
Done

Reply:
Hello @Ghiftee ,

WE own you an apologise. The dataypes of your columns were correct on your first attempt!

The main reason for this problem is that we had a mistake with the guidelines for code reviewers. We discussed it with the group of code reviewers and fixed it but a majority of the reviews that were performed before that discussion were done with the "safe mode" - by using the guidelines as a pattern.

We really apologize for the inconvenience that it caused. Good job with using correct datatypes at the beginning ❤️

We really recommend you to change them back to the initial state (in the schema.sql file but also in your database (for that please use ALET TABLE command)).